src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -202,45 +203,62 @@ private void DispatchSynchronizeTextChanges(
     {
         _ = _threadingContext.JoinableTaskFactory.RunAsync(async () =>
         {
-            var client = await RemoteHostClient.TryGetClientAsync(_workspace, _shutdownToken).ConfigureAwait(false);
-            if (client == null)
-                return;
-
-            // this pushes text changes to the remote side if it can. this is purely perf optimization. whether this
-            // pushing text change worked or not doesn't affect feature's functionality.
-            //
-            // this basically see whether it can cheaply find out text changes between 2 snapshots, if it can, it will
-            // send out that text changes to remote side.
-            //
-            // the remote side, once got the text change, will again see whether it can use that text change information
-            // without any high cost and create new snapshot from it.
-            //
-            // otherwise, it will do the normal behavior of getting full text from VS side. this optimization saves
-            // times we need to do full text synchronization for typing scenario.
-            if (!oldDocument.TryGetText(out var oldText) ||
-                !newDocument.TryGetText(out var newText))
+            var ableToSync = false;
```

Comment:
consider just having a helper method (or local function) that just returns a bool.